test/pummel/test-net-timeout.js

```diff
@@ -79,7 +79,7 @@ process.on('exit', function() {
   assert.ok(starttime != null);
   assert.ok(timeouttime != null);
 
-  diff = timeouttime - starttime;
+  var diff = timeouttime - starttime;
```

Comment:
Strange that both common and `'use strict'` didn't throw an error for this variable that wasn't previously declared. Glad it's fixed, but bothers me that wasn't caught before. Any thoughts as to why?